resources/views/discord/speedtest-threshold.blade.php

```diff
@@ -1,5 +1,9 @@
 **Speedtest Threshold Breached - #{{ $id }}**
 
+@if (!empty($mentionUser))
```

Comment:
Not required but you could also use the new `@blank` and `@filled` directives merged in #1951.